evalml/automl/automl_search.py

```diff
@@ -791,10 +811,12 @@ def search(self, show_iteration_plot=True):
             show_iteration_plot (boolean, True): Shows an iteration vs. score plot in Jupyter notebook.
                 Disabled by default in non-Jupyter enviroments.
         """
+        self._activate_logger()
         if self._searched:
-            logger.info(
+            self.logger.error(
                 "AutoMLSearch.search() has already been run and will not run again on the same instance. Re-initialize AutoMLSearch to search again."
             )
+            self._deactivate_logger()
```

Comment:
What would happen if we deleted `_deactivate_logger` entirely from this PR?

Reply:
Deleting `_deactivate_logger` would mean we can no longer run AutoML search with `verbose=False` after running with `verbose=True`, since our `get_logger` function (run when verbose is True) modifies the same logger that's grabbed by `logging.getLogger` when verbose is False.

Just now in reading your comments I had a thought - it doesn't have to be the case that the logger obtained when verbose is True vs when verbose is False have the same name. Instead, we could create another sub-logger for when verbose is True (like I did with `pipeline.describe()` and others) so that it would maintain any user-configured logging behavior without needing to be reset with `_deactivate_logger` at the end of each method that uses it. This would also resolve your question above about setting the logging level back to `logging.WARNING` in `_deactivate_logger`, as that would no longer be necessary. Thoughts?